src/modules/75-cd/strings/strings.en.yaml

```
@@ -366,6 +366,14 @@ serviceDashboard:
     infras: Infras
     clusters: '{{$.common.clusters}}'
     instances: '{{$.instanceFieldOptions.instances}}'
+    rollbackStatus: Rollback Status
+  postProdRollback:
+    rollbackTitle: Rollback from Which Infrastructure?
+    rollbackWarningText: 'Please make sure that you have a rollback steps/strategy defined. Otherwise rollback would fail, proceed to rollback?'
```

Please get this confirmed, the current phrase is incorrect
```suggestion
    rollbackWarningText: 'Please make sure that you have rollback steps/strategy defined. Otherwise, the rollback would fail. Do you want to proceed with rollback?'
```

---

Done